<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryTesterWidget</class>
 <widget class="QWidget" name="QueryTesterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="m_queryEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="m_checkTermGlobbing">
     <property name="text">
      <string>Enable query term globbing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_checkFilenamePattern">
     <property name="text">
      <string>Enable file name pattern detection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="m_sparqlView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
